### ISSUE TITLE
Need some help mapping WASD as arrow keys for SK61

### ISSUE THREAD

**Issue by dougcarneiro** [pixeltris/GK6X#157]
Hi,
I'm trying to make this:
[Base,Layer1]
LShift:TempSwitchFnLayer1

[FnLayer1]
W:Slash
A:RAlt
S:Menu
D:RCtrl

So I can have easy access to arrow keys when pressing LShift.
But it doenst seem to map properly.
What am I doing wrong? Im on Linux, lauching the program by calling mono on sudo. It connects to my keyboard without errors but I cant map it.

**Comment by Jonavin** [comment]
Why wouldn't you map it to the actual key you want?

[FnLayer1]
W:Up
A:Left
S:Down
D:Right

**Comment by dougcarneiro** [comment]
> Why wouldn't you map it to the actual key you want?
> 
> [FnLayer1]
> W:Up
> A:Left
> S:Down
> D:Right

It almost worked. For some reason I presumed that I was stucked to the keys I had on my keydump command.

I changed the config to this:
[Base,FnLayer1]
LShift:TempSwitchFnLayer2

[FnLayer2]
W:Up
A:Left
S:Down
D:Right

Unfortunately, holding LShift to access FnLayer2 is not working. If I switch to Layer2 manually on the keyboard and hold FN key, WASD work as arrow keys. But I really wanted to do that while holding LShift.

Sorry for being this dumb, still trying to learn that stuff :)
